src/pages/index.js

```diff
@@ -8,12 +8,34 @@ import Section from "../components/Section.js";
 
 import PopupWithForm from "../components/PopupWithForm.js";
 
-import { Settings, cardData } from "../Utils/Constants.js";
+import { Settings } from "../Utils/Constants.js";
 
 import PopupWithImage from "../components/PopupWithImage.js";
 
 import UserInfo from "../components/UserInfo.js";
 
+import Api from "../components/Api.js";
+
+// API
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "9a2a6717-e342-435d-a8a9-13aa291b9ad5",
+    "Content-Type": "application/json",
+  },
+});
+
+// Show and hide saving overlay
+function showSavingText(button) {
+  button.textContent = "Saving...";
+  button.disabled = true;
+}
+
+function hideSavingText(button) {
+  button.textContent = "Save";
+  button.disabled = true;
```

Comment:
If you disable the button here, then you can't click it in case of a server error


Please, enable the button in `catch` if you disable in `hideSavingText`